kvbc/test/sparse_merkle_storage/db_adapter_unit_test.cpp

```diff
@@ -749,10 +749,10 @@ TEST_P(db_adapter_custom_blockchain, delete_last_reachable_block) {
   for (auto i = 1; i <= numBlocks - 1; ++i) {
     ASSERT_EQ(adapter.addBlock(getOffsetUpdates(i, numBlocks)), i);
   }
-
+  int64_t updates_raw_data = 0;
```

Comment:
Maybe we can call this differently, i.e. `blockUpdatesByteSize` or similar. Additionally, I think it might be an unsigned type.